chuckie.py

```diff
@@ -1,22 +1,23 @@
 import random
+import boto3
 
-wotdOptions = {
-    "Spanish": "Hola",
-    "French": "Bonjour",
-    "English": "Hello",
-    "Texan": "Howdy",
-    "Chinese": "Ni Hao",
-    "Japanese": "Konichiwa"
-}
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table('wotd')
 
 def lambda_handler(event, context):
-    wotd = random.choice(list(wotdOptions.values()))
+    word = table.get_item(
+        Key={
+            'language': 'spanish',
+            'id': random.randrange(1, 697)
```

Comment:
Consider instead of fixing size of database at 697, to use function to retrieve size of the table.